src/corehost/cli/deps_resolver.cpp

```diff
@@ -151,68 +340,21 @@ pal::string_t deps_resolver_t::resolve_coreclr_dir(
             trace::verbose(_X("Deps has no coreclr entry."));
         }
```

Comment:
rid specific lookup for portable fat if deps cotnains it and probe fails to contain.